### DISCUSSION TITLE
Q&A for development

### DISCUSSION THREAD

**Discussion by dj1ch** [dj1ch/minigotchi#31] [Q&A]
Ask me questions, and I'll get back to you ASAP

**Comment by macgyverfuerarme** [comment]
How about using the Screen and the battery of a Flipper Zero and using the wemos as addon Board. I built an addon board for my flipper with the d1 Mini to use the spacehuhn deauth...
Are there any plans to implement a screen?

**Reply by dj1ch** [reply]
maybe. the problem with the d1 is that the pins are not soldered. if i solder them, i could totally implement that. check the to do on the readme. however, my plan as of right now is to have the output go through the terminal. it would be quick(er) and easier.

**Comment by smoochiee** [comment]
> Ask me questions, and I'll get back to you ASAP

Is this working?tried to flash it and its not deauthing..

**Reply by dj1ch** [reply]
not 100%, the frames being sent for deauths aren't valid by the IEEE 802.11 standards. im working on fixing those frames soon. the projects mostly broken for the most part but a new release planned to release sometime soon will probably be the solution to all our problems. it doesn't help either that my development machine(laptop) stopped working and i need to fix it.

**Reply by smoochiee** [reply]
ok bro..ill be following this project...any chance you can add a whitelist SSID selection via screen menu...

**Reply by dj1ch** [reply]
> ok bro..ill be following this project...any chance you can add a whitelist SSID selection via screen menu...

if i can solder the pins to a d1 mini, and get a screen, i sure can. i might settle for a web ui, and have a network you can connect to and adjust settings and check logs. its a lot of things to haul and i thank you for the support
